src/Global/molecules/UserMenu.tsx

```diff
@@ -144,6 +144,11 @@ const FooterText = styled.div`
   text-overflow: ellipsis;
   user-select: none;
   white-space: nowrap;
+  max-width: 136px;
```

Comment:
Make this conditional if there is a icon then it should be 136px else it should be 161px.
Please verify this on all resolutions.